aim/db/models.py

```diff
@@ -18,16 +18,67 @@
 from aim.db import model_base
 
 
+class Tenant(model_base.Base, model_base.HasName,
+             model_base.HasDisplayName, model_base.AttributeMixin):
+    """DB model for Tenant."""
+
+    __tablename__ = 'aim_tenants'
+
+
 class BridgeDomain(model_base.Base, model_base.HasName,
+                   model_base.HasDisplayName, model_base.HasTenantNameKey,
                    model_base.AttributeMixin):
     """DB model for BridgeDomain."""
 
     __tablename__ = 'aim_bridge_domains'
 
-    tenant_name = sa.Column(sa.String(64), primary_key=True)
-    vrf_name = sa.Column(sa.String(64))
+    vrf_name = model_base.name_column()
     enable_arp_flood = sa.Column(sa.Boolean)
     enable_routing = sa.Column(sa.Boolean)
     limit_ip_learn_to_subnets = sa.Column(sa.Boolean)
     l2_unknown_unicast_mode = sa.Column(sa.String(16))
     ep_move_detect_mode = sa.Column(sa.String(16))
+
+
+class Subnet(model_base.Base, model_base.HasTenantNameKey,
+             model_base.AttributeMixin):
+    """DB model for Subnet."""
+
+    __tablename__ = 'aim_subnets'
+
+    bd_name = model_base.name_column(primary_key=True)
```

Comment:
Same consideration as above about FKs, should (tenant_name, bd_name) point to an actual BD?